cou/utils/juju_utils.py

```diff
@@ -329,6 +329,17 @@ async def _get_supported_apps(self) -> list[str]:
             name for name, app in model.applications.items() if is_charm_supported(app.charm_name)
         ]
 
+    async def get_unit(self, name: str) -> JujuUnit:
```

Comment:
Wouldn't be easier and cleaner transform `_get_unit` into `get_unit`?

Reply:
I shouldn't do that because it's a breaking refactor change. It shouldn't happen in this PR.